packages/workspace/src/tasks-runner/utils.ts

```diff
@@ -152,3 +149,19 @@ function unparseOption(key: string, value: any, unparsed: string[]) {
     unparsed.push(`--${key}=${value}`);
   }
 }
+
+function interpolate(template: string, data: any): string {
+  template = template.replace(/{([\s\S]+?)}/g, function (match: string) {
+    let value = data;
+    let path = match.slice(1, -1).trim().split('.').slice(1);
+    for (let idx = 0; idx < path.length; idx++) {
+      console.log(value[path[idx]]);
+      if (!value[path[idx]]) throw new Error(`Could not replace {${match}}!`);
```

Comment:
I'd throw a more concrete error: Could not interpolate output ...

Reply:
Done